### PR TITLE
Gh 142 support different db name

### PR DESCRIPTION
**Pull request by at055612**
Addresses #142

### CODE REVIEW

File: stroom-auth-svc/src/main/java/stroom/auth/service/App.java [hunk 1]
```diff
@@ -63,6 +63,11 @@
     private static final org.slf4j.Logger LOGGER = org.slf4j.LoggerFactory.getLogger(App.class);
 
     public static final String SESSION_COOKIE_NAME = "authSessionId";
+
+    // This is the schema/database name used for jooq code generation and in integration tests.
+    // Runtime schema mapping is used to allow us to run with a different DB/schema name
+    //private static final String INTERNAL_DB_NAME = "auth";
+
```
Comment (jc064522) [inline stroom-auth-svc/src/main/java/stroom/auth/service/App.java]: Commented out?

File: stroom-auth-svc/src/main/java/stroom/auth/service/App.java [hunk 2]
```diff
@@ -121,6 +129,19 @@ public void run(Config config, Environment environment) throws Exception {
...
+//                    config.getDatabaseName(), INTERNAL_DB_NAME);
+//
+//            jooqConfig.settings()
+//                    .withRenderMapping(new RenderMapping()
+//                            .withSchemata(new MappedSchema()
+//                                    .withInput(INTERNAL_DB_NAME)
+//                                    .withOutput(config.getDatabaseName())));
+//        }
+//    }
+
```
Comment (jc064522) [inline stroom-auth-svc/src/main/java/stroom/auth/service/App.java]: Commented out?

File: stroom-auth-svc/src/main/java/stroom/auth/service/App.java (hunk 1 above)
Comment (at055612) [reply stroom-auth-svc/src/main/java/stroom/auth/service/App.java]: Removed in commit below

File: stroom-auth-svc/src/main/java/stroom/auth/service/App.java (hunk 2 above)
Comment (at055612) [reply stroom-auth-svc/src/main/java/stroom/auth/service/App.java]: Removed in commit below

**Pull request merged**